consumer.py

```diff
@@ -128,7 +128,9 @@ def _set_halted_to_zero():
 def _message_failed(msg):
     global c
     message_class = _get_class_from_topic_name(msg.topic())
-    if Configuration.ACK_ON_FAIL:
+    if Configuration.ACK_ON_FAIL is True or (
+        type(Configuration.ACK_ON_FAIL) == list and message_class.base_name in Configuration.ACK_ON_FAIL
```

Comment:
i alway get confused on what to choose when, between type and isinstance.
i read that isinstance is better in the case of python class. 
i know it won't differ much, but just wanted to know if there is something that type provides.

Reply:
Thanks for this information. I don't think it matters in this case because there are no classes but I think it makes the code more readable so I will change the code to use it here.